docs/docs/40-contributor-guide/30-code-of-conduct.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Code of conduct
-description: Code of conduct
+sidebar_label: Code of conduct
+description: Contributor Code of Conduct for the open source Kargo project
 ---
 
-# Contributor Covenant Code of Conduct
+# Kargo Contributor Code of Conduct
```

Comment:
```suggestion
# Contributor Covenant Code of Conduct
```

Reply:
Oh, I didn't realize that `Covenant` was [purposeful](https://www.contributor-covenant.org/version/2/1/code_of_conduct/code_of_conduct.md).